docs/README.md

```diff
@@ -1,94 +1,31 @@
-
-### Feast does not _fully_ solve
-* **reproducible model training / model backtesting / experiment management**: Feast captures feature and model metadata, but does not version-control datasets / labels or manage train / test splits. Other tools like [DVC](https://dvc.org/), [MLflow](https://www.mlflow.org/), and [Kubeflow](https://www.kubeflow.org/) are better suited for this.
-* **batch feature engineering**: Feast supports on-demand and streaming transformations. Feast is also investing in supporting batch transformations. 
-* **native streaming feature integration:** Feast enables users to push streaming features, but does not pull from streaming sources or manage streaming pipelines.
-* **lineage:** Feast helps tie feature values to model versions, but is not a complete solution for capturing end-to-end lineage from raw data sources to model versions. Feast also has community contributed plugins with [DataHub](https://datahubproject.io/docs/generated/ingestion/sources/feast/) and [Amundsen](https://github.com/amundsen-io/amundsen/blob/4a9d60176767c4d68d1cad5b093320ea22e26a49/databuilder/databuilder/extractor/feast\_extractor.py). 
-* **data quality / drift detection**: Feast has experimental integrations with [Great Expectations](https://greatexpectations.io/), but is not purpose built to solve data drift / data quality issues. This requires more sophisticated monitoring across data pipelines, served feature values, labels, and model versions.
-
-## Example use cases
-
-Many companies have used Feast to power real-world ML use cases such as:
-
-* Personalizing online recommendations by leveraging pre-computed historical user or item features.
-* Online fraud detection, using features that compare against (pre-computed) historical transaction patterns
-* Churn prediction (an offline model), generating feature values for all users at a fixed cadence in batch
-* Credit scoring, using pre-computed historical features to compute the probability of default
-
-## How can I get started?
-
-{% hint style="info" %}
-The best way to learn Feast is to use it. Head over to our [Quickstart](getting-started/quickstart.md) and try it out!
-{% endhint %}
-
-Explore the following resources to get started with Feast:
-
-* [Quickstart](getting-started/quickstart.md) is the fastest way to get started with Feast
-* [Concepts](getting-started/concepts/) describes all important Feast API concepts
-* [Architecture](getting-started/architecture/) describes Feast's overall architecture.
-* [Tutorials](tutorials/tutorials-overview/) shows full examples of using Feast in machine learning applications.
-* [Running Feast with Snowflake/GCP/AWS](how-to-guides/feast-snowflake-gcp-aws/) provides a more in-depth guide to using Feast.
-* [Reference](reference/feast-cli-commands.md) contains detailed API and design documents.
-* [Contributing](project/contributing.md) contains resources for anyone who wants to contribute to Feast.
+## Core Components
```

Comment:
i think this piece may be a little confusing as the components we also have in the documentation don't list the Python SDK (https://docs.feast.dev/getting-started/components) but it does have the feature server.